Editor/Resources/EditorWindow/Pages/AnywherePage.uxml

```diff
@@ -30,7 +30,13 @@
         <ui:Foldout name="AnywherePageLaunchTitle" class="foldout separator separator--vertical" text="Launch foldout">
             <ui:VisualElement class="form-row">
                 <ui:Label name="AnywherePageConfigureClientLabel" class="form-row__label" text="Configure Client"/>
-                <ui:Label name="AnywherePageConfigureClientDescription" class="form-row__text-display" text="Description"/>
+                <ui:VisualElement class="form-row__input">
+                    <ui:Button name="AnywherePageConfigureClientButton" class="button button--primary" text="Configure button"/>
```

Comment:
As discussed offline, this button and the Launch server button should be primary conditionally (same conditions that enable/disable them)

Reply:
I think my latest changes takes care of this